### ISSUE TITLE
Expanding the ALTERNATE_DOMAINS functionality 

### ISSUE THREAD

**Issue by manedfolf** [tootsuite/mastodon#8921]
I'd like to propose a feature that would allow users to choose how their username domain appears on their profile when an instance has multiple domains specified in ALTERNATE_DOMAINS.

For example, my instance Yiff.Life also has fur.lgbt and furry.church set as alternate domains, and currently when someone searches up @sky@fur.lgbt, @sky@yiff.life is what resolves. 

It would be an interesting extension of ALTERNATE_DOMAINS if a user had a drop down in their profile settings so that they could change what appears as their default domain, although it could include on their profile that their main instance is the primary LOCAL_DOMAIN.

Thoughts?

* * * *

- [X] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by joelburns** [comment]
I would love to see this happen. It's 4 years later… any chance of this?

**Comment by Volkor3-16** [comment]
> I would love to see this happen. It's 4 years later… any chance of this?

https://github.com/mastodon/mastodon/issues/2668 seems like the more active issue for this

**Comment by manedfolf** [comment]
I disagree that the issues are exactly the same, although there is significant overlap in implementation. #2668 seems to be specific to individual users. This is regarding an instance hosting and presenting multiple domains while still having a core/primary domain for identity integrity.

**Comment by Volkor3-16** [comment]
Oh definitely not the same, but similar enough. (and considering the amount of activity/who created it, it seems more likely)
